src/population/visualization/family_tree_tracker.py

```diff
@@ -0,0 +1,201 @@
+from datetime import datetime
+import json
+import networkx as nx
+import os
+import numpy as np
+from loguru import logger
+from population.visualization.visualization import create_and_save_figure, make_dir_if_not_exists
+from population.visualization.graphing_model import get_neural_net_positions
+from population.visualization.gene_data_processing import convert_genes_to_numerical, get_pca_positions, get_pca_colors
+
+class FamilyTreeTracker:
+    """
+    A class used to track relations between nodes, along with other attributes.
+    This saves the genome data as a series of lines in a temporary file, and
+    """
+
+    def __init__(self, temp_file_dir: str='temp_genome_data', delete_temp_file: bool=True):
```

Comment:
You should create a config class for this class and add these default values there.